### ISSUE TITLE
Menu options in Membership sites

### ISSUE THREAD

**Issue by PlanetThoughts** [Simplero/Roadmap#278]
Currently, it seems that course names spread across the top of a Membership site. The following change is relatively simple to make. I will also be looking at the tab-to-a-page option that Siif mentioned.

Since there may be many courses, it would be good to have the option to have all courses appear in a single dropdown instead of spreading in multiple rows across the top of the Membership site. If there are courses for which a person does not have access, it would be good to have a sub-option to either show those courses and if clicked give an opportunity to buy a product, or the other option, only show courses for which the person already has access. This would apply to both the horizontal and the dropdown version of the course listing.

**Comment by teresa-slp** [comment]
I agree that the member site navigation is a bit cumbersome. While setting up my not-yet-functioning member site, I've tried to get the top nav words very short because they spread out so much on the top as PlanetThoughts describes above.

Furthermore, I have 11 modules in each of my programs. I have postponed creating the rest of my programs in the member site because the side bar navigation in the "courses" is nightmarishly long and I've barely begun!

Please see attached photos.  Please, is there a way to simplify this? On my current WP site, I have dropdowns coming from the top like a regular website and it seems to work well.

<img width="1075" alt="simplero member site 1" src="https://user-images.githubusercontent.com/31087371/39552501-5c5687ba-4e1e-11e8-8286-38cde46b41ac.png">
<img width="961" alt="simplero member site 2" src="https://user-images.githubusercontent.com/31087371/39552502-5c6e6902-4e1e-11e8-9159-7e960233753e.png">
<img width="1312" alt="wordpress nav bar on top" src="https://user-images.githubusercontent.com/31087371/39552503-5c82ff7a-4e1e-11e8-9e56-21d79d22152e.png">

**Comment by Grethe1** [comment]
Thanks David for requesting this - I would also really love a better overview of my Memberships sites and not having to scroll down to find the last one - that's always the newest site lowest... because the boxes are so big... do they need to be that huge?

Thanks Calvin!! Wihuuu!!! Grethe

**Comment by Ryan-Henderson** [comment]
This is similar to #394 / #379 / #353
I am into a drop-down menu feature for sure!

**Comment by danielidacademy** [comment]
Absolutely me too! I was about to make the same request - but there is already 5 other requests regarding it! Thanks and great! :)

**Comment by emilyannpeterson** [comment]
Plus 1 for a drop-down menu option in membership sites!
